apps/web/src/services/asset-inventory/components/CloudServiceAdmin.vue

```diff
@@ -65,88 +55,72 @@ export default {
         },
     },
     setup(props) {
-        const state = reactive({
+        const storeState = reactive({
             users: computed<UserReferenceMap>(() => store.getters['reference/userItems']),
+        });
+        const state = reactive({
             fields: [
                 { label: 'User ID', name: 'user_id' },
-                { label: 'User Name', name: 'resource_id' },
-                { label: 'Role', name: 'role_info.name' },
-                { label: 'Labels', name: 'labels' },
+                { label: 'User Name', name: 'user_name' },
             ],
-            items: [],
+            projectUserIdList: [] as string[],
+            refinedItems: computed<UserItem[]>(() => {
+                const users: UserItem[] = state.projectUserIdList.map((d) => ({
+                    user_id: d,
+                    user_name: storeState.users[d]?.name ?? d,
+                }));
+                return users.filter((d) => {
+                    const searchText = state.searchText.toLowerCase();
+                    return d.user_id.toLowerCase().includes(searchText)
+                        || d.user_name.toLowerCase().includes(searchText);
+                });
+            }),
             loading: true,
             totalCount: 0,
-            options: {
-                sortBy: 'user_id',
-                sortDesc: true,
-                pageStart: 1,
-                pageLimit: 15,
-                searchText: '',
-            },
+            searchText: '',
+            pageLimit: 15,
+            pageStart: 1,
             timezone: computed(() => store.state.user.timezone ?? 'UTC'),
         });
 
-        const apiQuery = new ApiQueryHelper();
-        const getQuery = () => apiQuery.setSort(state.options.sortBy, state.options.sortDesc)
-            .setPage(
-                state.options.pageStart,
-                state.options.pageLimit,
-            )
-            .setFilters([
-                { v: state.options.searchText },
-            ])
-            .data;
-
-        const api = SpaceConnector.client.identity.project.member.list;
-        const listAdmin = async () => {
+        const getProjectUserData = async () => {
             state.loading = true;
-
             try {
-                const res = await api({
-                    include_parent_member: true,
+                const params: ProjectGetRequestParams = {
+                    workspace_id: '', // TODO: workspace_id
```

Comment:
suggestion: I think it's not needed because workspace_id will be added by SpaceConnector level.